api/util.py

```diff
@@ -150,6 +168,12 @@ def _update_db(db, datainfo):
             new=True,
             projection=PROJECTION_FIELDS,
             )
+    try:
+        session_label = session['label']
+    except KeyError:  # this can happen if nims_metadata_status == None
+        session_label = '(unknown)'
+    log.info('Using group_id="%s", project_name="%s", and session_label="%s"'
+             % (project['group'], project['name'], session_label))
```

Comment:
You can always open a PR into my branch with changes, it might work better
than silently committing. I could also give you write permissions to my
fork if you'd prefer to avoid opening PRs.